src/hotspot/share/gc/shared/gcUtil.cpp

```diff
@@ -86,10 +86,10 @@ void AdaptivePaddedAverage::sample(float new_sample) {
 
   // Now update the deviation and the padded average.
   float new_avg = average();
-  float new_dev = compute_adaptive_average(fabsd(new_sample - new_avg),
+  float new_dev = compute_adaptive_average((float)fabsd(new_sample - new_avg),
```

Comment:
Is JDK-6302670 still relevant, and/or are there other reasons we avoid `cmath`? This could just be `fabsf`

Reply:
JDK-630267 does not seem relevant.  We should clean that up (or maybe at least the comment).  I changed these occurrences to fabs.